src/main/java/com/google/api/codegen/transformer/java/JavaFeatureConfig.java

```diff
@@ -19,22 +19,26 @@
 import com.google.api.codegen.config.MethodContext;
 import com.google.api.codegen.config.ResourceNameMessageConfigs;
 import com.google.api.codegen.config.ResourceNameType;
+import com.google.api.codegen.config.TransportProtocol;
 import com.google.api.codegen.transformer.DefaultFeatureConfig;
 import com.google.auto.value.AutoValue;
 
 @AutoValue
 public abstract class JavaFeatureConfig extends DefaultFeatureConfig {
+  @Override
+  public abstract boolean resourceNameTypesEnabled();
 
   @Override
-  public abstract boolean enableStringFormatFunctions();
```

Comment:
The deleted lines are just cleanup?

Reply:
This is GitHub diff tool doing poor job =(. Nothing was removed, just shifted a bit. This method is still there on line 29 in the "new" version of the file, shown as addition.

Reply:
oops. I missed that!